insteon/api.py

```diff
@@ -55,6 +55,36 @@ def delete(self, path, data={}):
         response = requests.delete(API_URL + path, headers=self._set_headers())
         return self._check_response(response, self.delete, path, data)
 
+    def stream(self, path, devices_to_watch={}):
+        headers = self._set_headers()
+        headers['Content-Type'] = 'text/event-stream'
+        try:
+            while True:
+                response = requests.get(API_URL + path, headers = headers, stream=True)
+                for line in response.iter_lines():
+                    # filter out keep-alive new lines
+                    if line:
+                        decoded_line = line.decode('utf-8')
+                        payload = decoded_line.split(': ')
+                        self._handle_stream_message(payload[0], payload[1], devices_to_watch)
+
+        except APIError as e:
+            print(e)
```

Comment:
As per logging. Not really. I haven't in other places so I think I'll make that a separate release.

APIError's are usually because streams can't be opened but I will make sure to defer a close on the connection!